src/model/entities/rundown.ts

```diff
@@ -676,7 +676,12 @@ export class Rundown extends BasicRundown {
   }
 
   public reset(): void {
+    const rundownModePriorToDeactivation: RundownMode = this.mode
     this.deactivate()
+    if (rundownModePriorToDeactivation === RundownMode.REHEARSAL) {
+      this.enterRehearsal()
+      return
+    }
     this.activate()
```

Comment:
You can now use void functions in ternaries, so you could write it like this:
```
const isRundownInRehearsalBeforeResetting: boolean = this.mode === RundownMode.REHEARSAL
this.deactivate()
isRundownInRehearsalBeforeResetting ? this.enterRehearsal() : this.activate()
```

Reply:
Using void in ternaries should not be something new?